.github/workflows/drc_switch_checking_test.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a Github Actions for testing DRC rule deck of GF180MCU
+name: DRC Switch Checking test
```

Comment:
I don't know what a "Switch Checking test" is, so most other people will not either. Please use a more descriptive name.

Reply:
The DRC rule deck has some switches (Like a script with arguments) that tell it what to check specifically.
I'm not sure what a proper name could be used.

Reply:
I added (Rule deck arguments) to its name.